builtin/logical/aws/backend.go

```diff
@@ -60,6 +65,7 @@ func Backend() *backend {
 		Invalidate:        b.invalidate,
 		WALRollback:       b.walRollback,
 		WALRollbackMinAge: minAwsUserRollbackAge,
+		PeriodicFunc:      b.rotateExpiredStaticCreds,
```

Comment:
I think we'll have some problems if this is used in the context of a Vault enterprise cluster (performance standbys, secondaries, or disaster recovery). Each Vault instance will run this periodic func and rotate credentials in AWS. I imagine there will be read-only storage errors as well.

I suggest we prevent this by only running this on the active instance in the primary cluster and on local mounts. An example of how this is checked can be found at https://github.com/hashicorp/vault/blob/main/builtin/logical/database/rotation.go#L521-L528.

Reply:
done